scout/server/blueprints/institutes/static/form_scripts.js

```diff
@@ -1,26 +1,3 @@
-var searchTerm = document.getElementById('search_term');
```

Comment:
Moved to constants/cases/case_tags, so it ca be used by both cases and dashboard pages